pkg/webhook/merge_on_review_approval_and_status_checks.go

```diff
@@ -40,27 +40,41 @@ func (h *autoMerger) HandleEvent(w http.ResponseWriter, payload interface{}, ghC
 		return h.handlePullRequestEvent(w, event, ghClientFunc)
 	case *github.StatusEvent:
 		return h.handleStatusEvent(w, event, ghClientFunc)
+	case *github.PullRequestReviewEvent:
+		return h.handlePullRequestReviewEvent(w, event, ghClientFunc)
 	default:
 		return nil
 	}
 }
 
+func (h *autoMerger) handlePullRequestReviewEvent(w http.ResponseWriter, event *github.PullRequestReviewEvent, ghClientFunc GitHubIntegrationsClientFunc) error {
```

Comment:
It's safe to try to merge something twice, only one will succeed. Guess I could check merge status of PR in this case, but due to async nature of webhooks not sure that will protect against multiple attempts to merge PR, bit of a race condition.